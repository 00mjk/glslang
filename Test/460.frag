#version 460 core

struct S {
    float f;
    vec4 v;
};

in S s;

void main()
{
    interpolateAtCentroid(s.v);
}
