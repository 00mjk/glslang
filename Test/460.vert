#version 460 core

int i;
; // extraneous semicolon okay
float f;;;

void main()
{
}
;
;
