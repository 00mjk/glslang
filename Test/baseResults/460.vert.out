460.vert
Shader version: 460
0:? Sequence
0:7  Function Definition: main( ( global void)
0:7    Function Parameters: 
0:?   Linker Objects
0:?     'i' ( global int)
0:?     'f' ( global float)
0:?     'gl_VertexID' ( gl_VertexId int VertexId)
0:?     'gl_InstanceID' ( gl_InstanceId int InstanceId)


Linked vertex stage:


Shader version: 460
0:? Sequence
0:7  Function Definition: main( ( global void)
0:7    Function Parameters: 
0:?   Linker Objects
0:?     'i' ( global int)
0:?     'f' ( global float)
0:?     'gl_VertexID' ( gl_VertexId int VertexId)
0:?     'gl_InstanceID' ( gl_InstanceId int InstanceId)

